README.md

```diff
@@ -1,2 +1,95 @@
 # moveit_example_apps
-Demonstrator applications for using MoveIt with full robotic setups
+
+This project demonstrates advanced applications for using MoveIt with full robotic setups. For each listed application, there is a tutorial introducing how to replicate and run the application either in simulation or with real robot. If you are interested in the technique details, you can visit [MoveIt Tutorials](https://ros-planning.github.io/moveit_tutorials/) for further information.
+
+## Applications
+
+| Application Name | Documentation          | Docker |
+| ---------------- | -----------------------| ------ |
+| Random Pick      | [random_pick_tutorial] | WIP    |
+
+[random_pick_tutorial]: https://roboticsyy.github.io/moveit_example_apps/doc/random_pick/random_pick.html
+
+## Continuous Integration
+
+Work in progress.
```

Comment:
Can we do this now, using moveit_ci? Its not too difficult to setup, you can probably just copy the configuration from https://github.com/ros-planning/moveit_grasps/blob/melodic-devel/.travis.yml

Reply:
We can try to setup CI with moveit_ci. The difficulty lies in the dependencies of the app, that are compiled against caffe or OpenVINO, they all require multiple configurations.